vc-verifier/kotlin/vcverifier/src/main/java/io/mosip/vercred/vcverifier/credentialverifier/verifier/CwtVerifier.kt

```
@@ -0,0 +1,136 @@
+        val cleanHex = hex.replace("\\s".toRegex(), "")
+        require(cleanHex.length % 2 == 0) { "Invalid hex length" }
+
+        return ByteArray(cleanHex.length / 2) { i ->
+            cleanHex.substring(i * 2, i * 2 + 2).toInt(16).toByte()
+        }
+    }
+
+    private fun decodeCose(cwtBytes: ByteArray): CBORObject? {
+        return try {
+            CBORObject.DecodeFromBytes(cwtBytes)
+        } catch (e: Exception) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)